### PR TITLE
DOC: README update

### PR DESCRIPTION
**Pull request by ZviBaratz**
Organized a bit, removed some older references, added a bunch of badges.
If and when approved, I will copy the contents to [`info.py`](https://github.com/nipy/nibabel/blob/master/nibabel/info.py).

[PREVIEW](https://github.com/ZviBaratz/nibabel/tree/doc/readme#readme)

### CODE REVIEW

**Review by effigies — COMMENTED** [review]
Thanks, Zvi! I made a batch of suggestions, but this is overall a tremendous improvement.

As always, feel free to argue back if I have been insufficiently persuasive...

**Comment by ZviBaratz** [comment]
I'm glad you approve, thank you for all the fixes and suggestions. 
Looking at it now, I feel like the "Packaging" section should be removed. Will suggest a fix tomorrow.

File: README.rst [hunk 1]
```diff
@@ -1,110 +1,143 @@
...
+         :target: https://codecov.io/gh/nipy/NiBabel
+         :alt: codecov badge
+
+   * - PyPI
+     -
+      .. image:: https://img.shields.io/pypi/v/nibabel.svg
+         :target: https://pypi.python.org/pypi/nibabel/
+         :alt: PyPI version
+      .. image:: https://img.shields.io/pypi/dm/nibabel.svg
+         :target: https://pypistats.org/packages/nibabel/
```
Comment (effigies) [inline README.rst:70]: 404 error

```suggestion
         :target: https://pypistats.org/packages/nibabel
```

File: README.rst [hunk 2]
```diff
@@ -1,110 +1,143 @@
...
+         :alt: Debian Unstable package
+      .. image:: https://repology.org/badge/version-for-repo/aur/python:nibabel.svg?header=Arch%20%28%41%55%52%29
+         :target: https://repology.org/project/python:nibabel/versions
+         :alt: Arch (AUR)
+      .. image:: https://repology.org/badge/version-for-repo/gentoo_ovl_science/nibabel.svg?header=Gentoo%20%28%3A%3Ascience%29
+         :target: https://repology.org/project/nibabel/versions
+         :alt: Gentoo (::science)
+      .. image:: https://repology.org/badge/version-for-repo/nix_unstable/python:nibabel.svg?header=nixpkgs%20unstable
+         :target: https://repology.org/project/python:nibabel/versions
+         :alt: nixpkgs unstable
```
Comment (effigies) [inline README.rst:86]: I still think we should include conda.

```suggestion
   * - Packages
     -
     .. image:: https://img.shields.io/conda/vn/conda-forge/nibabel
         :target: https://anaconda.org/conda-forge/nibabel
         :alt: Conda package
      .. image:: https://repology.org/badge/version-for-repo/debian_unstable/nibabel.svg?header=Debian%20Unstable
         :target: https://repology.org/project/nibabel/versions
         :alt: Debian Unstable package
      .. image:: https://repology.org/badge/version-for-repo/aur/python:nibabel.svg?header=Arch%20%28%41%55%52%29
         :target: https://repology.org/project/python:nibabel/versions
         :alt: Arch (AUR)
      .. image:: https://repology.org/badge/version-for-repo/gentoo_ovl_science/nibabel.svg?header=Gentoo%20%28%3A%3Ascience%29
         :target: https://repology.org/project/nibabel/versions
         :alt: Gentoo (::science)
      .. image:: https://repology.org/badge/version-for-repo/nix_unstable/python:nibabel.svg?header=nixpkgs%20unstable
         :target: https://repology.org/project/python:nibabel/versions
         :alt: nixpkgs unstable
```

**Review by effigies — APPROVED** [review]
LGTM. Two small comments.

**Comment by ZviBaratz** [comment]
@effigies re-requested one last review to make sure copying to [`long_description`](https://github.com/nipy/nibabel/blob/625c75bb8efa640d87f4b9c6b8af168c9bc36462/nibabel/info.py#L14) was done correctly. 

Also, @matthew-brett, is this OK with you?

**Comment by effigies** [comment]
Some of this doesn't look great in the docs. I'll push a suggestion.

image

**Comment by effigies** [comment]
Updated:

image

**Comment by effigies** [comment]
We're good to go if you're happy with my changes.

**Pull request merged**